### PR TITLE
Prepare configuration and data for docusaurus-based blog

### PR DESCRIPTION
**Pull request by Fashander**
# Description

In this PR we split config for blog and config for docs.
We also introduce new docker container in docker-compose to run a blog instance.
We add new tasks to the Taskfile to build/run those containers.

The deployment config for blog is set [here](https://github.com/ferretdb/blog). We tried it, it worked.

To try this locally, you can run `task blog-dev`, and then open http://127.0.0.1:3001

## Readiness checklist

* [ ] I added unit tests for new functionality or bug fixes.
* [ ] I added integration tests for new functionality or bug fixes.
* [ ] I added compatibility tests for new functionality or bug fixes.
* [ ] I made spot refactorings.
* [ ] I updated user documentation.
* [ ] I ran `task all`, and it passed.
* [ ] I added/updated comments for both exported and unexported top-level declarations (functions, types, etc).
* [ ] I checked comments rendering with `task godocs`.
* [x] I ensured that the title is good enough for the changelog.
* [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Assignee, Labels, Project and project's Sprint fields.
* [x] I marked all done items in this checklist.

### CODE REVIEW

**Comment by rumyantseva** [comment]
@Fashander if you don't mind, I'll push github actions related code to this PR too (that's for my tomorrow's todo list as it's not a blocker at the moment).

**Comment by Fashander** [comment]
@rumyantseva Sure, no problem.

File: website/blog/2022-06-27-developers-need-ferretdb-stackoverflow-developer-survey-2022.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+**So what is the solution to MongoDB's vendor lock-in?**
+
+At FerretDB, we are working on combining the best of two worlds: the robustness and freedom of PostgreSQL, and the unmatched developer experience of MongoDB.
+
+How we interpret the StackOverflow Developer Surveyfrom the perspective of FerretDB is that we are on the right track: we are working on a solution which is based on PostgreSQL, and we are providing an open source alternative to MongoDB.
+Those service providers or companies currently running PostgreSQL will be able to cater to the needs of developers seeking for the ease of use of MongoDB
+
+However, we are not going to stop at PostgreSQL: we are planning to support more databases, so developer experience can be standardized across many different platforms.
+
+Working with partners, like [Tigris Data](https://tigrisdata.io), we can even provide an alternative to MongoDB Atlas itself.
```
Comment (b1ron) [inline website/blog/2022-06-27-developers-need-ferretdb-stackoverflow-developer-survey-2022.md:50]:
```suggestion
Working with partners, like [Tigris Data](https://www.tigrisdata.com/), we can even provide an alternative to MongoDB Atlas itself.
```

**Review by b1ron — CHANGES_REQUESTED** [review]
Looks good and it's running locally. Link to https://tigrisdata.io/ is dead, I suggested a fix.

File: website/blog/2022-06-27-developers-need-ferretdb-stackoverflow-developer-survey-2022.md (hunk 1 above)
Comment (rumyantseva) [reply website/blog/2022-06-27-developers-need-ferretdb-stackoverflow-developer-survey-2022.md:50]: Nice catch!

**Review by b1ron — APPROVED** [review]

**Review by w84thesun — APPROVED** [review]

**Review by noisersup — DISMISSED** [review]
:rocket:

**Review by rumyantseva — COMMENTED** [review]
Almost ready! We need to move the recent blog post, and then we should be ready to publish the new blog!

**Comment by chilagrow** [comment]
> Almost ready! We need to move the recent blog post, and then we should be ready to publish the new blog!

`FerretDB v0.8.0 Beta` blog is missing.

**Review by rumyantseva — DISMISSED** [review]
LGTM for now.

We will continue in the next PRs and issues.

**Review by w84thesun — DISMISSED** [review]

**Review by chilagrow — DISMISSED** [review]

**Review by AlekSi — APPROVED** [review]

**Review by rumyantseva — APPROVED** [review]
LGTM

**Pull request merged**